lib/gringotts/gateways/sagepay.ex

```diff
@@ -34,68 +23,71 @@ defmodule Gringotts.Gateways.Sagepay do
+  your account "secrets" in the `Sub-Accounts > Overview` section.
+
+  Here's how the secrets map to the required configuration parameters for SagePay:
+
+  | Config parameter        | SagePay secret         |
+  | ------------------------| -----------------------|
+  | `:auth_id`              | **Authorization Id**   |
+  | `:vendor_name`          | **Name of a merchant** |
+
+  Your Application config **must include the `:Authorization Id`, `:Merchant name`
+  fields** and would look something like this:
+
+      config :gringotts, Gringotts.Gateways.SagePay,
+        auth_id: "your_secret_user_id",
+        vendor_name: "name_of_vendor"
+        
 
   ## Scope of this module
 
-  > It's unlikely that your first iteration will support all features of the
-  > gateway, so list down those items that are missing.
+  * SagePay does not process money in cents, and the `amount` is converted to integer.
+  * SagePay supports payments from various cards and banks.
 
   ## Supported currencies and countries
 
-  > It's enough if you just add a link to the gateway's docs or FAQ that provide
-  > info about this.
+      :AUD, :CAD, :CHF, :CYP, :DKK, :EUR, :GBP, :HKD, :INR, :JPY, 
+      :MTL, :NOK, :NZD, :RUB, :SEK, :SGD, :THB, :TRY, :USD, :ZAR 
 
   ## Following the examples
 
-  1. First, set up a sample application and configure it to work with MONEI.
+  1. First, set up a sample application and configure it to work with SagePay.
   - You could do that from scratch by following our [Getting Started][gs] guide.
       - To save you time, we recommend [cloning our example
       repo][example] that gives you a pre-configured sample app ready-to-go.
           + You could use the same config or update it the with your "secrets"
-          as described [above](#module-registering-your-monei-account-at-sagepay).
-
-  2. Run an `iex` session with `iex -S mix` and add some variable bindings and
-  aliases to it (to save some time):
-  ```
-  iex> alias Gringotts.{Response, CreditCard, Gateways.Sagepay}
-  iex> card = %CreditCard{first_name: "Jo",
-                          last_name: "Doe",
-                          number: "4200000000000000",
-                          year: 2099, month: 12,
-                          verification_code: "123", brand: "VISA"}
-  ```
+          as described [above](#module-registering-your-sagepay-account-at-gringotts).
 
-  > Add any other frequently used bindings up here.
+  2. To save a lot of time, create a [`.iex.exs`][iex-docs] file as shown in
+     [link][sagepay.iex.exs] to introduce a set of handy bindings and
+     aliases.
 
   We'll be using these in the examples below.
 
-  [gs]: https://github.com/aviabird/gringotts/wiki/
-  [home]: https://www.sagepay.co.uk/
-  [example]: https://github.com/aviabird/gringotts_example
+  [iex-docs]: https://hexdocs.pm/iex/IEx.html#module-the-iex-exs-file
+  [sagepay.iex.exs]: https://github.com/Anwar0902/graph/blob/master/sagepay.iex.exs
```

Comment:
:heart: :yellow_heart: :purple_heart: Thanks for making a `.iex.exs`! There are many ways of sharing it with others, and I highly recommend making a [gist](http://gist.github.com/) instead of a file in a repo.